lib/stats/processors/student_activity.rb

```diff
@@ -0,0 +1,195 @@
+# encoding: UTF-8
+module Stats
+  module Processors
+    class StudentActivity < ProcessorCore
+
+      sheet_name "Aktivita studentov"
+
+      sheet_split "D3"
+
+      CREDENTIALS = ["AISID", "Login", "Priezvisko", "Meno", ]
+      METRICS_HEADER = ["Návštevy (uč.)",       #"Počet zobrazených otázok",
+                        "Návštevy (test.)",     #"Počet zobrazených správnych odpovedí",
+                        "Evaluator",            #"Počet hodnotení otázok type Evaluation",
+                        "Správne (unikátne)",   #"Počet doteraz správne zodpovedaných otázok",
+                        "Správne",              #"Počet doteraz správne zodpovedaných pokusov",
+                        "Nesprávne",            #"Počet doteraz nesprávne zodpovedaných pokusov",
+                        "Neviem",               #"Počet kliknutí na odpoveď Neviem",
+                        "Pozn."                 #"Počet vložených poznámok"
+                       ]
+
+      def self.week_start(setup, week)
+        setup.first_week_at.to_date + week.number * 7 - 7
+      end
+
+      def self.get_weeks(setup)
+        right_weeks = []
+        all_weeks = setup.weeks.order(:number).reverse_order
+        all_weeks.each do |aw|
+          #puts "WEEK (aw): #{aw.id} , #{aw.number} WEEK_END: #{(week_start(setup,aw)+6)} TIME_NOW: #{Time.now.to_date}"
+          if (week_start(setup,aw)) <= Time.now.to_date
+            #puts "WEEK_ID: #{aw.id}"
+            right_weeks << aw
+          end
+        end
+        return right_weeks
+      end
+
+      def self.process(setup)
+
+        # TODO
+        # Testovacia verzia
+        su = User.all
+        su_map = su.map(&:id)
+        users = User.where(id: su_map)
```

Comment:
Toto vyberie vsetkych pouzivatelov z DB, od kazdeho vyberie `id` a potom hlada pouzivatelov podla `id` (cize opat vsetkych). To cele sa da nahradit `users = User.all`.